facebook/create-react-app
# Upgrade to Jest 25

Fixes https://github.com/facebook/create-react-app/issues/8361, fixes #8689, fixes #7491.

### QA

I hope that CI will ensure that Jest 25 works. 

But I'm wondering what is the best way to try link react-script into an existing application and try it out? I'll try to verify that the suggested changes will work with the CRA-powered apps that I'm currently working on.
